packages/governance/test/unitTests/test-fakeContractGovernor.js

```diff
@@ -0,0 +1,237 @@
+import { test } from '@agoric/zoe/tools/prepare-test-env-ava.js';
+import '@agoric/zoe/exported.js';
+
+import { makeNotifierFromAsyncIterable } from '@agoric/notifier';
+import { makeZoeKit } from '@agoric/zoe';
+import bundleSource from '@endo/bundle-source';
+import buildManualTimer from '@agoric/zoe/tools/manualTimer.js';
+import { makeFakeVatAdmin } from '@agoric/zoe/tools/fakeVatAdmin.js';
+import { E } from '@endo/eventual-send';
+
+import { resolve as importMetaResolve } from 'import-meta-resolve';
+import { MALLEABLE_NUMBER } from '../swingsetTests/contractGovernor/governedContract.js';
+import { CONTRACT_ELECTORATE, ParamTypes } from '../../src/index.js';
+
+const governedRoot = '../swingsetTests/contractGovernor/governedContract.js';
+const contractGovernorRoot = '../../tools/fakeContractGovernor.js';
+const autoRefundRoot = '@agoric/zoe/src/contracts/automaticRefund.js';
+
+const makeBundle = async sourceRoot => {
+  const url = await importMetaResolve(sourceRoot, import.meta.url);
+  const path = new URL(url).pathname;
+  const contractBundle = await bundleSource(path);
+  return contractBundle;
+};
+
+// makeBundle is a slow step, so we do it once for all the tests.
+const contractGovernorBundleP = makeBundle(contractGovernorRoot);
+const governedBundleP = makeBundle(governedRoot);
+// could be called fakeCommittee. It's used as a source of invitations only
```

Comment:
+1 to the naming the thing by what it does instead of what it it's intended to be used for (and then a comment referencing the use). Is there such a name for `fakeContractGovernor`? Consider `directContractGovernor` or `puppetContractGovernor`.

Reply:
I re-read the terminology guides on `Fakes`, `Mocks` and `Stubs`, and of those, `Fake` seemed clearly the most appropriate. I wouldn't mind `puppetContractGovenor`, however.

Reply:
Agree that among those `Fake` is most (and only) appropriate option. I maintain the "puppet" would be better because it describes the way in which it is fake (how it differs from what isn't fake) but I won't withhold approval on that.

Reply:
renamed.